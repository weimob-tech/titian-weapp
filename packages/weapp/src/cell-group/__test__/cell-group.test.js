const { getComponent } = require('../../common/test/getComponent');

describe('test component', () => {
  it('should render success', () => {
    const comp = getComponent('cell-group');
    expect(comp).toMatchSnapshot();
  });

  it('should render title、subTitle success', () => {
    const comp = getComponent('cell-group', {
      title: '标题',
      subTitle: '副标题'
    });
    expect(comp).toMatchSnapshot();
  });

  it('should render different mode success', () => {
    const comp = getComponent('cell-group', {
      mode: 'card'
    });

    expect(comp).toMatchSnapshot();
  });
});
