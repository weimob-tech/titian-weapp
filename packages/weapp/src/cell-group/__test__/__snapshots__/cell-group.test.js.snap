// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component should render different mode success 1`] = `
<main>
  <wx-view
    class="ext-class titian-cell-group"
  >
    <wx-view
      class="titian-cell-group-body titian-cell-group-card"
    />
  </wx-view>
</main>
`;

exports[`test component should render success 1`] = `
<main>
  <wx-view
    class="ext-class titian-cell-group"
  >
    <wx-view
      class="titian-cell-group-body"
    />
  </wx-view>
</main>
`;

exports[`test component should render title、subTitle success 1`] = `
<main>
  <wx-view
    class="ext-class titian-cell-group"
  >
    <wx-view
      class="titian-cell-group-title-wrap title-wrap"
    >
      <wx-text
        class="titian-cell-group-title title-class"
      >
        标题
      </wx-text>
      <wx-text
        class="titian-cell-group-sub-title sub-title-class"
      >
        副标题
      </wx-text>
    </wx-view>
    <wx-view
      class="titian-cell-group-body"
    />
  </wx-view>
</main>
`;
